doc/example-nginx-blockdaemon.conf

```diff
@@ -14,31 +14,49 @@ http {
 
   client_max_body_size 0;
 
-  upstream bdnodes {
-    server YOUR_NODE.bdnodes.net:443;
-    server YOUR_NODE.bdnodes.net:443;
-    server YOUR_NODE.bdnodes.net:443;
-    keepalive 4;
+  # Pythnet servers
+  server {
+    listen 7900;
+    server_name pythnet.localhost;
+    location / {
+      proxy_set_header Upgrade $http_upgrade;
+      proxy_set_header Connection "Upgrade";
+      proxy_ssl_server_name on;
+      proxy_set_header Host "rpc.mainnet.pythnet.blockdaemon.tech";
+      proxy_pass https://rpc.mainnet.pythnet.blockdaemon.tech:443/websocket;
+    }
+  }
+  server {
+    listen 7899;
+    server_name pythnet.localhost;
+    location / {
+      proxy_http_version 1.1;
+      proxy_set_header Connection "";
+      proxy_ssl_server_name on;
+      proxy_set_header Host "rpc.mainnet.pythnet.blockdaemon.tech";
+      proxy_pass https://rpc.mainnet.pythnet.blockdaemon.tech:443/;
```

Comment:
there's no token auth on the pythnet blockdaemon rpc nodes? (I presume this is their public RPC endpoint?)

Reply:
The Pythnet endpoints are public.